ci/debian-gorgone/control

```diff
@@ -1,7 +1,7 @@
 Source: centreon-gorgone
 Section: net
 Priority: optional
-Maintainer: Luiz Costa <me@luizgustavo.pro.br>
+Maintainer: Centreon <contact@centreon.com>
```

Comment:
This file is no more useful.
We should remove all the files about gorgoned in centreon-collect.
It already has been done in the develop branch but obviously not here.